apps/web/lib/supabase-queries.ts

```diff
@@ -148,27 +148,6 @@ export const queries = {
 // MARK: - Mutations
 
 export const mutations = {
-  useDeletePipeline: () =>
-    useMutation(
-      async ({
-        pipelineId,
-        resourceId,
-      }: {
-        pipelineId: Id['pipe']
-        resourceId: Id['reso']
-      }) => {
-        await browserSupabase.from('pipeline').delete().eq('id', pipelineId)
-        // TODO: Need to properly handle this on the server to
-        // 1) Remove orphan resource
-        // 2) Revoke before deleting as needed
-        if (resourceId.includes('debug')) {
-          await browserSupabase.from('resource').delete().eq('id', resourceId)
-        }
-      },
-      // TODO: Revoke & remove the resources as pipelines are deleted
-      // We should have a background job for this probably...
-    ),
-
```

Comment:
Moved to `ConnectionCard.tsx` so we can define `onSuccess`, `onSettled` without needing to duplicate `useMutation` API.

Reply:
![CleanShot 2023-02-20 at 07 37 39@2x](https://user-images.githubusercontent.com/696842/219982343-519809aa-3caa-4d0d-8226-778f7ce4616c.png)
Btw you can actually define `onSettled` etc. when calling the mutate function too. Just gotta pass in an additional set of options.